app/src/main/kotlin/com/hedvig/app/feature/checkout/CheckoutViewModel.kt

```diff
@@ -87,7 +81,7 @@ class CheckoutViewModel(
     val checkout = model.checkout
     when (checkout?.status) {
       Checkout.CheckoutStatus.PENDING -> _events.trySend(Event.Loading)
-      Checkout.CheckoutStatus.SIGNED -> createAccessToken()
+      Checkout.CheckoutStatus.SIGNED -> offerRepository.fetchNewOffer(quoteCartId)
```

Comment:
Fast forwarded to the success case of createAccessToken() since we should not be creating this token no more from the checkout.
As discussed before if we're inside the checkout we must already be logged in regardless.